docker/lnd/Dockerfile

```diff
@@ -24,6 +24,12 @@ RUN : "${COMMIT_SHA:?COMMIT_SHA Build argument needs to be set.}"
 
 RUN git clone https://github.com/lightningnetwork/lnd . \
 &&  git checkout ${COMMIT_SHA} \
+# Cherrypick a dep change that replaces `git.schwanenlied.me/yawning/bsaes.git`
+# with a mirror so we dont have to depend on external hosts for third-party
+# dependencies
+&&  git config --global user.email "dev@sparkswap.com" \
+&&  git config --global user.name "dev" \
```

Comment:
Why do we need `email` and `name` to cherry-pick?

Reply:
`git cherry-pick` will grab the commit from the commit SHA and then record a new commit (applied with the current user). If we dont include email and name git will yell at us that our client is not configured yet